draft-ietf-httpbis-digest-headers.md

```diff
@@ -150,7 +150,7 @@ data and HTTP content.
 
 This document defines the `Representation-Digest` request and response header
 and trailer field ({{representation-digest}}) that contains a digest value
-computed by applying a hashing algorithm to `selected representation data`
+computed by applying a hashing algorithm to *selected representation data*
```

Comment:
I think we'll end up revisiting how best to draw attention to these terms. Emboldening doesn't seem like something the RFC editor would do but we can cross that bridge later on.

Reply:
Sure, I already filed an issue on kramdown for that ;)

Reply:
https://github.com/cabo/kramdown-rfc2629/issues <-